server/src/main/java/org/elasticsearch/common/util/concurrent/EsExecutors.java

```diff
@@ -125,14 +125,32 @@ public static EsThreadPoolExecutor newFixed(
         ThreadFactory threadFactory,
         ThreadContext contextHolder,
         boolean trackExecutionTime
+    ) {
+        return newFixed(
+            name,
+            size,
+            queueCapacity,
+            threadFactory,
+            contextHolder,
+            trackExecutionTime ? TaskTrackingConfig.DEFAULT : TaskTrackingConfig.DO_NOT_TRACK
+        );
+    }
+
+    public static EsThreadPoolExecutor newFixed(
```

Comment:
Instead of adding an alternate signature, could you please update the existing uses of newFixed? It seems like a trivial update to the signature (can be done with intellij refactoring tools), and there are only 21 uses I see (almost all for tests).